### PR TITLE
Fix suppliers resource

### PR DESCRIPTION
**Pull request by enricostano**
```
GET /api/v1/suppliers?group_id=27
```
**Response body:** an array with all the suppliers related to the group with id `27`
**Status code:** `200`
**Permissions:** only admins and members of group with id 27 can perform this request
```
GET /api/v1/suppliers/2
```
**Response body:** the supplier with id `2`
**Status code:** `200`
**Permissions:** only admins and members of the group associated with the requested supplier can perform this request
```
POST /api/v1/suppliers
{
  group_id: 27,
  producer_id: 134
}
```
**Response body:** the created resource
**Status code:** `201`
**Permissions:** only admins of group with id `27` can perform this request
```
DELETE /api/v1/suppliers/2
```
**Response body:** empty body
**Status code:** `204`
**Permissions:** only admins and members of the group associated with the requested supplier can perform this request

### CODE REVIEW

File: engines/suppliers/app/controllers/suppliers/api/v1/suppliers_controller.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            group: @group,
+            producer: @producer
+          )
+
+          if supplier.save
+            render json: SupplierSerializer.new(supplier)
+          else
+            render(
+              status: :bad_request,
+              json: supplier.errors
```
Comment (enricostano) [inline engines/suppliers/app/controllers/suppliers/api/v1/suppliers_controller.rb]: `error:` in json response

Comment (enricostano) [inline engines/suppliers/app/controllers/suppliers/api/v1/suppliers_controller.rb]: Yep... we just need more time :bowing_man:

File: db/seeds.rb [hunk 2]
```diff
@@ -1,17 +1,73 @@
-user = ::Account::User.create(
+# Users
+
+joanin = ::Account::User.create!(
```
Comment (sauloperez) [inline db/seeds.rb]: lol

File: db/seeds.rb [hunk 3]
```diff
@@ -1,17 +1,73 @@
...
   email: 'joanin@katuma.org',
   first_name: 'Joanin',
   last_name: 'Cerea',
   username: 'joanin',
   password: 'kakaka22'
 )
 
-group = ::BasicResources::Group.create(
+frida = ::Account::User.create!(
```
Comment (sauloperez) [inline db/seeds.rb]: do we need the `::`? This is the global namespace

File: db/seeds.rb [hunk 4]
```diff
@@ -1,17 +1,73 @@
...
+  basic_resource_group_id: cabas.id,
   role: ::BasicResources::Membership::ROLES[:admin]
 )
+
+# Producers
+
+jaume = ::BasicResources::Producer.new(
+  name: 'El Jaume',
+  email: 'jaume@eljaume.coop',
+  address: 'c/ de les peras, 33, Barbera del Valles'
```
Comment (sauloperez) [inline db/seeds.rb]: > geo localization

Uf... do we need PostGIS already? :trollface:

File: engines/suppliers/app/controllers/suppliers/api/v1/suppliers_controller.rb (hunk 1 above)
Comment (andresgutgon) [inline engines/suppliers/app/controllers/suppliers/api/v1/suppliers_controller.rb]: This is wrong. You need to add an `error` key

Comment (enricostano) [inline engines/suppliers/app/controllers/suppliers/api/v1/suppliers_controller.rb]: read my first comment, but yes... will need to fix it

File: db/seeds.rb [hunk 5]
```diff
@@ -1,17 +1,73 @@
-user = ::Account::User.create(
+# Users
+
+joanin = ::Account::User.create!(
   email: 'joanin@katuma.org',
   first_name: 'Joanin',
   last_name: 'Cerea',
   username: 'joanin',
   password: 'kakaka22'
```
Comment (andresgutgon) [inline db/seeds.rb]: I'm having problems getting used to this `k` thing. Can we change by `p`. Seriously is annoying :)

Comment (enricostano) [inline db/seeds.rb]: LOL, ok...

**Pull request merged**